inst/eDISH_app/modules/renderSettings/renderSettings.R

```diff
@@ -215,6 +215,15 @@ renderSettings <- function(input, output, session, data, settings, status){
 #      }
 #    }
 
+    
```

Comment:
Seems like we might want to move this to generate settings, no?

Reply:
@jwildfire Great idea :)